lib/utils/ModuleBase.js

```diff
@@ -42,4 +44,16 @@ export default class ModuleBase {
   get app(): App {
     return this._app;
   }
+
+  get serviceUrl(): ?string {
+    return this._serviceUrl;
+  }
+
+  /**
+   * Returns the a key that uniquely identifies the native instance for this module
+   * @return {string}
+   */
+  get nativeModuleKey(): string {
```

Comment:
This was in `utils/native.js` to prevent internals from polluting the public API.

Unfortunately Flow doesn't support the concept of `@private` without having to write a separate definitions file.

This is something we might do in the future, if we have time, but for the time being it would be good if we can move this back into `utils/native.js` and also see if there's a way to hide the `serviceUrl` getter as well.